### ISSUE TITLE
Add ability to ignore certain results

### ISSUE THREAD

**Issue by jpzwarte** [dequelabs/axe-core#4707]
### Product

axe-core

### Feature Description

The color-contrast rule sometimes outputs `Element's background color could not be determined because it is overlapped by another element` messages. These messages are a result of how the CSS & DOM is structured and not something that is easily "fixed". Instead i would like the ability to ignore these specific messages. I do NOT want to disable the entire color-contrast rule. Just these results.

Perhaps
```js
ignoreResults: `Element's background color could not be determined because it is overlapped by another element`
```

Or
```js
ignoreResults: /(overlapped by another element)/
```

**Comment by WilcoFiers** [comment]
I think that's a reasonable proposal. Axe already has an option to allow setting failed results to be incomplete (needs review) instead. I think something that makes all incomplete results of one rule an automatic pass would make sense to me.
